### PR TITLE
Navigation Link Changes #558

### LINKED ISSUE #558: Navigation Link Changes
We need to do following changes in the client side: 

- [ ] Change password on Sidebar of Dashboard

- [ ] Add "Participate in Challenges" button next to the "Host Competition" button on main page

- [ ] Replace "Host Competition" with "Host Challenge" on Home Page

- [ ] Show © CloudCV 2016-2017  on footer

### PR DESCRIPTION
**Pull request by gauthamzz**
Fixed #558 

Update password , add participate in challenges , add @ in footer and changed to host challenge 
screenshot from 2017-01-27 04-24-06

screenshot from 2017-01-27 04-28-35
screenshot from 2017-01-27 04-28-40
screenshot from 2017-01-27 04-28-44

@deshraj review , i hope it is as per you said

### CODE REVIEW

**Comment by deshraj** [comment]
@gauthamzz all looks good except that we need some different icon for "Participate in Challenges" and "Change Password". @aka-jain thoughts?

**Comment by gauthamzz** [comment]
should this [line[(https://github.com/Cloud-CV/EvalAI/blob/master/frontend/src/js/route-config/route-config.js#L248) , [this](https://github.com/Cloud-CV/EvalAI/blob/master/frontend/src/views/web/host-challenge.html#L5)  be changed also ??

**Comment by aka-jain** [comment]
@gauthamzz : Please redirect to challenge LIst page when user clicks on participate challenge button.
Also use this icon for participation "fa-free-code-camp".
Make participation button color white with black color text. Write a seperate styles for the button.
We already have chage Password view, please redirect it to same on clicking change password link.
In footer, change color of copyright so that it can be visible but use color lighter than other links in footer for copyright. Do these fixes please.

**Comment by gauthamzz** [comment]
@aka-jain its written host competition shouldnt it be host challenge ?

**Comment by aka-jain** [comment]
Oh Okay, don't do it for now. Actually we are going to revamp the host challenge view soon and no need to change it now. Will do it later.

**Comment by aka-jain** [comment]
Updating again:
@gauthamzz : Please redirect to challenge LIst page when user clicks on participate challenge button.
Also use this icon for participation "fa-free-code-camp".
Make participation button color white with black color text. Write a seperate styles for the button.
We already have chage Password view, please redirect it to same on clicking change password link.
In footer, change color of copyright so that it can be visible but use color lighter than other links in footer for copyright. Do these fixes please.

**Comment by gauthamzz** [comment]
@aka-jain is it okay now ?

File: frontend/src/views/web/partials/sidebar.html [hunk 1]
```diff
@@ -84,6 +88,10 @@
                         <i class="fa fa-pencil side-link-icon"></i> Update Profile</a>
                 </li>
                 <li>
+                    <a class="waves-effect waves-dark text-light-gray ev-md-container" ui-sref="web.change-password" ui-sref-active="side-active-link">
```
Comment (aka-jain) [inline frontend/src/views/web/partials/sidebar.html]: @gauthamzz Please make this indent.

File: frontend/src/views/web/partials/sidebar.html [hunk 2]
```diff
@@ -84,6 +88,10 @@
...
+                        <i class="fa fa-pencil side-link-icon"></i> Change Password</a>
```
Comment (aka-jain) [inline frontend/src/views/web/partials/sidebar.html]: This too.

**Comment by deshraj** [comment]
@gauthamzz can you complete this before working on other issues so that I can merge it?

File: frontend/src/views/web/partials/sidebar.html (hunk 1 above)
Comment (gauthamzz) [inline frontend/src/views/web/partials/sidebar.html]: its already indented right :confused:

Comment (deshraj) [inline frontend/src/views/web/partials/sidebar.html]: The `<a>` tag is unindented with the next `<i>`.

Comment (deshraj) [inline frontend/src/views/web/partials/sidebar.html]: My bad. Looks like you are right. May be @aka-jain meant something else.

Comment (gauthamzz) [inline frontend/src/views/web/partials/sidebar.html]: @deshraj I have no clue :confused:

Comment (deshraj) [inline frontend/src/views/web/partials/sidebar.html]: Let's wait for him to reply on this thread. :-)

File: frontend/src/views/web/landing.html [hunk 3]
```diff
@@ -8,6 +8,11 @@
 </section>
 <!-- EvalAI into -->
 <section class="ev-container center">
+	<h4 class="ev-head w-300 text-white">EvalAI</h4>
```
Comment (aka-jain) [inline frontend/src/views/web/landing.html]: @gauthamzz Please indent this. Reduce tab.

File: frontend/src/views/web/partials/footer.html [hunk 4]
```diff
@@ -4,11 +4,17 @@
...
+                    <ul class="inline-list">
+                        <li><a class="text-light-gray" href="#!"><i class="fa fa-github"></i></a></li>
+                        <li><a class="text-light-gray" href="#!"><i class="fa fa-twitter"></i></a></li>
+                        <li><a class="text-light-gray" href="#!"><i class="fa fa-facebook"></i></a></li>
+                    </ul>
+                <p><a class="text-white">© CloudCV 2016-2017</a></p>
+                    <ul class="inline-list">
+                        <li><a class="text-light-gray" href="#!"><i class="fa fa-github"></i></a></li>
+                        <li><a class="text-light-gray" href="#!"><i class="fa fa-twitter"></i></a></li>
+                      <li><a class="text-light-gray" href="#!"><i class="fa fa-facebook"></i></a></li>
```
Comment (aka-jain) [inline frontend/src/views/web/partials/footer.html]: I think you have replicated this code. Please check line 7 and 13

Comment (aka-jain) [inline frontend/src/views/web/partials/footer.html]: Indent this too.

File: frontend/src/views/web/partials/sidebar.html (hunk 1 above)
Comment (aka-jain) [inline frontend/src/views/web/partials/sidebar.html]: Oh sorry my bad, haven't seen the closing anchor tag.

File: frontend/src/views/web/landing.html [hunk 5]
```diff
@@ -10,6 +10,11 @@
 <section class="ev-container center">
     <h4 class="ev-head w-300 text-white">EvalAI</h4>
     <p class="text-light-gray ev-intro w-300">Be a leader in Computational Research</p>
+    <br>
+    <a ng-click="main.hostChallenge($event)" class="waves-effect waves-dark btn ev-btn-dark w-300">Host Challenge&nbsp; <i class="fa fa-paper-plane-o text-light-gray icon"></i></a>
+    <a ui-sref="web.challenge-main" class="waves-effect waves-dark btn ev-btn-light text-dark-black w-300">Participate in Challenges&nbsp; <i class="fa fa-free-code-camp text-dark-black icon"></i></a>
+    <h4 class="ev-head w-300 text-white">EvalAI</h4>
```
Comment (aka-jain) [inline frontend/src/views/web/landing.html]: Please check line line 11 and 16, again replication. Try to check all things in your local server.

Comment (gauthamzz) [inline frontend/src/views/web/landing.html]: it came up when i solved merge conflicts !

**Comment by deshraj** [comment]
@aka-jain can you review this so that we can merge?

**Pull request merged**